routers/bpa_register.py

```diff
@@ -77,7 +103,17 @@ async def register_bpa_user(
                     status_code=400,
                     detail=f"Registration failed: {response.json()['message']}",
                 )
-            return {"message": "User registered successfully", "user": response.json()}
+
+        if bpa_resources:
+            email_service = EmailService()
```

Comment:
Let's set up the email service in the background task - i'm not sure how expensive/slow it is, but there's no reason to do it here.